src/Content/Translation.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace Kirby\Content;
+
+use Kirby\Cms\Helpers;
+use Kirby\Cms\Language;
+use Kirby\Cms\ModelWithContent;
+
+/**
+ * Each page, file or site can have multiple
+ * translated versions of their content,
+ * represented by this class
+ *
+ * @package   Kirby Content
+ * @author    Bastian Allgeier <bastian@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ */
+class Translation extends ContentTranslation
+{
+	/**
+	 * Creates a new translation object
+	 */
+	public function __construct(
+		protected ModelWithContent $model,
+		protected Version $version,
+		protected Language $language
+	) {
+	}
+
+	/**
+	 * Improve `var_dump` output
+	 * @codeCoverageIgnore
+	 */
+	public function __debugInfo(): array
+	{
+		return $this->toArray();
+	}
+
+	/**
+	 * Returns the language code of the
+	 * translation
+	 *
+	 * @deprecated since 5.0.0 Use `::language()->code()` instead
```

Comment:
`Todo`: Consistent syntax with existing uses:

```suggestion
	 * @deprecated 5.0.0 Use `$translation->language()->code()` instead
```